### ISSUE TITLE
Add WASM publication

### ISSUE THREAD

**Issue by ArnyminerZ** [gabrieldrn/carbon-compose#24]
Hi, I'm trying to use the library, and I've seen that theoretically it supports WASM, which is awesome, but I can't get it to import with Gradle. I'm getting the following error:
```
:composeApp:wasmJsMain: Could not resolve io.github.gabrieldrn:carbon:0.1.0.
Required by:
    project :composeApp

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```
And I've seen that it's not showing on Maven: https://mvnrepository.com/artifact/io.github.gabrieldrn

Was there an issue when publishing the release? Or is it not supported yet?

Thanks for the library by the way 🚀

**Comment by gabrieldrn** [comment]
Hello, thank you for reporting this issue!
Actually the wasm demo has been setup right after I've published the current version - which doesn't have a configured wasm target... my bad for writing that it is available in the documentation website.
I still need to setup the publication for the wasm target, it will be configured for the next release.
Until then, feel free to clone the project and setup + build your own artifacts for your app.
Sorry for the inconvenience.

**Comment by ArnyminerZ** [comment]
Okay, no problem, just wanted to give a heads-up in case it was forgotten or something like that 😉

**Comment by gabrieldrn** [comment]
The Wasm artifact is now available with version 0.2.0 😉

https://central.sonatype.com/artifact/io.github.gabrieldrn/carbon-wasm-js
